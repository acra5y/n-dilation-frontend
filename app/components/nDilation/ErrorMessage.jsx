import React from "react";

function ErrorMessage() {
    return <div>Ooops, something went terribly wrong ðŸ¤¯</div>;
}

export default ErrorMessage;
