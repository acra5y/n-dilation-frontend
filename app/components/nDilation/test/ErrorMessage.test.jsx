import React from "react";
import { shallow } from "enzyme";

import ErrorMessage from "../ErrorMessage";

const render = () => shallow(<ErrorMessage />);

describe("ErrorMessage", () => {
    it("should render", () => {
        const component = render();

        expect(component).toMatchSnapshot();
    });
});
