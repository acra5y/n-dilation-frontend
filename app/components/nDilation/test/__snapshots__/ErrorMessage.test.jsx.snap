// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ErrorMessage should render 1`] = `
<div>
  Ooops, something went terribly wrong ðŸ¤¯
</div>
`;
